samples/dashboards/demo/datagrid-cell-format/demo.css

```
@@ -0,0 +1,45 @@
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
+
+#container {
+    height: 235px;
+    margin: 8px;
+}
+
+.high-color {
+    color: #3ea100;
+}
+
+.low-color {
+    color: #c11a1a;
```

```suggestion
    color: #c11a1a;
    font-weight: bold;
```
For better visibility of color changes